examples/nidcpower_source_dc_voltage/measurement.py

```diff
@@ -78,89 +83,97 @@ def measure(
         # Long measurements may require a longer timeout.
         timeout=60,
     ) as reservation:
-        if len(reservation.session_info) != 1:
+        if len(pin_names) != len(voltage_levels):
             measurement_service.context.abort(
                 grpc.StatusCode.INVALID_ARGUMENT,
-                f"Unsupported number of sessions: {len(reservation.session_info)}",
+                f"The voltage level input did not match the number of pins configured",
             )
```

Comment:
Single-session measurements like this one are the use case for the single-session helper function I have been suggesting.

NI-DCPower supports channel expansion, so I think it's reasonable for the NI-DCPower example to be a single-session measurement.

Reply:
Added two APIs as mentioned